README.md

```diff
@@ -47,6 +47,7 @@ You will need a plugin manager to install the plugins and prompts listed on this
 
 - [Official Documentation](http://fishshell.com/docs/current/index.html)
 - [fish cookbook](https://github.com/brj/fish-cookbook) - Tips and recipes for fish-shell users
+- [fish Tutorial](https://github.com/ellerbrock/tutorial-fish-shell-setup-osx) - How to Setup fish Shell with fisherman, Powerline Fonts, iTerm2 and Budspencer Theme on OS X.  
```

Comment:
@brj still awesome :) i just finished to write the shell script to automate the process of the installation of all the parts and tools like homebrew, fish shell, fisher, powerline fonts, iterms2, iterms2colorschemas, dependencies & tools. i think its pretty need instead of copy and pasting through the documentation. i updated the repo and added the script, and put the source in the readme file. hope you guys like it too ...